code/modules/goals/station_goals/bluespace_tap.dm

```diff
@@ -175,6 +175,13 @@
 		/obj/item/pizzabox,
 	)
 
+/obj/effect/spawner/lootdrop/bluespace_tap/mats
+	name = "exotic materials"
```

Comment:
Materials aren't really "exotic". In fact id say its the opposite.

Reply:
fine